### ISSUE TITLE
spaceAroundOperators adds space in front of "&" in pointer function calls

### ISSUE THREAD

**Issue by davbeck** [nicklockwood/SwiftFormat#138]
Probably easier to show than describe:

```swift
var red: CGFloat = 0
var green: CGFloat = 0
var blue: CGFloat = 0
var alpha: CGFloat = 0
hexColor.getRed(&red, green: &green, blue: &blue, alpha: &alpha)
```

When spaceAroundOperators is enabled, a space is added before `&red`, which is not what I would expect.

Granted, a better importer would probably move red into the argument list.

**Comment by nicklockwood** [comment]
@davbeck yep, I agree this is a bug. I'll add a fix for it.

**Comment by nicklockwood** [comment]
This is fixed on master. I'll include the fix in the next release.

**Comment by nicklockwood** [comment]
Fixed in 0.28.6.
